qa/automation/nikeev/Any Career Portal Page/Cookies.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Cookies settings on any career portal page
+
+Background: 'Necessary cookies' is a required option in Cookies Settings box
+
+Scenario Outline: User sees cookies pop-up on any portal page
+	Given user previously didn't accept cookies for career portal
+	When user opens <name of the page> page
+	Then user sees a cookie pop-up at the bottom of the screen
```

Comment:
Usually we describe in details what a user should see

Reply:
For all tests in this feature or for all tests in all features? And how detailed should the description be? For Then in this case I should write: 
Then user sees a cookie pop-up at the bottom of the screen
And pop-up contains a cookie picture, a message: "We use cookies to personalize content and improve your browsing experience." and two buttons: 'Accept and Continue' and ' Learn more'. 
Or something more or less detailed? 
And what about changes in visual implementation? Tests with a high level of fixed details (if they are not variables) will become invalid very easy